gfp_test.go

```diff
@@ -0,0 +1,157 @@
+package bn256
```

Comment:
Should files have CF copyright on them?

Reply:
the main license file is from the go original project, and I assume that applies to all the files in the repo.